onnxruntime/core/mlas/lib/quantize.cpp

```diff
@@ -263,6 +469,60 @@ Return Value:
     }
 }
 
+
+template<typename DataType>
+void
+MLASCALL
+MlasQLinearAddKernel(
+    const DataType* InputA,
+    float ScaleA,
+    DataType ZeroPointA,
+    const DataType* InputB,
+    float ScaleB,
+    DataType ZeroPointB,
+    float ScaleC,
+    DataType ZeroPointC,
+    DataType* OutputC,
+    size_t LengthA,
+    size_t LengthB
+    )
+{
+    constexpr int32_t MinimumValue = std::numeric_limits<DataType>::min();
+    constexpr int32_t MaximumValue = std::numeric_limits<DataType>::max();
+
+    size_t N = std::max(LengthA, LengthB);
+    if (LengthA == 1) {
+        float ValueA = ScaleA * (int(InputA[0]) - int(ZeroPointA));
+        for (size_t n = 0; n < N; n++) {
+            float ValueB = ScaleB * (int(InputB[n]) - int(ZeroPointB));
+            int32_t IntValueC = (int32_t)std::nearbyintf((ValueA + ValueB) / ScaleC) + int32_t(ZeroPointC);
+            IntValueC = std::max(IntValueC, MinimumValue);
+            IntValueC = std::min(IntValueC, MaximumValue);
+            OutputC[n] = (DataType)IntValueC;
+        }
+    }
+    else if (LengthB == 1) {
```

Comment:
"} else" instead of newline like other code

Reply:
fixed.